nodejs.spec.in

```diff
@@ -286,12 +286,16 @@ BuildRequires:  openssl-has-RSA_get0_pss_params
 Requires:       openssl-has-RSA_get0_pss_params
 %endif
 
-%if 0%{?suse_version} >= 1500 
+%if 0%{?suse_version} >= 1500
 BuildRequires:  openssl >= %{openssl_req_ver}
-%else
+%elif 0%{?suse_version}
```

Comment:
Thanks but sadly we cannot use `%elif` since this only works with rpm 4.17+ and we need this to work with SLE12.

Maybe it would make sense to just have a separate `%if 0%{?fedora_version}` outside the suse_version case?

Reply:
I will fix it. Thanks!